package.json

```diff
@@ -71,6 +71,7 @@
     "bootstrap-sass": "3.3.7",
     "chalk": "^1.1.3",
     "cheerio": "^0.22.0",
+    "child-process": "^1.0.2",
```

Comment:
This is possibly not required. `child-process` being used in `runWebpack.js` is included in the [node standard library](https://nodejs.org/docs/latest-v6.x/api/child_process.html).